app/models/customer.rb

```diff
@@ -105,14 +116,7 @@ def preferred_document_locale
   end
 
   def provider_customer
```

Comment:
I assume you're keeping this method because it's used all over the place?

Reply:
yeah, becasue there are quite a lot of places we're using it 🙈 
but actually i think declaring this method as alias will be better 😅 
thank you for putting my attention here